### ISSUE TITLE
tissue search

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#7659]
**Is your feature request related to a problem? Please describe.**

From a Curator:

<blockquote>

If you go to Arctos without logging in and search for "UAM:Mamm" and enter "tissue" in Parts, 45,251 returns. This, I believe, is what most non-Arctosians would reasonably expect to give them a reasonably accurate number. 

If you search for "UAM:Mamm" and are familiar enough with Arctos to know to customize and put "tissue=yes" (not a reasonable expectation, I would argue), 79,281 returns.

If you search VertNet--which has not added any new UAM mammal records for night on 2 or 3 years now and specify "UAM", "Mammalia", and "has tissues", estimated 112,409 returns.

A colleague (and fellow curator at another museum who is very well versed in these things) just pointed these discrepancies out to me and I am passing it along.

</blockquote>

**Describe what you're trying to accomplish** 

Facilitate users getting what they want.

**Describe the solution you'd like**

Not a clue....

**Describe alternatives you've considered**

Collections can direct their audience to a non-default search, eg https://arctos.database.museum/search.cfm?sp=preset_default&guid_prefix=UAM:Mamm&is_tissue=1 sets search terms, sets results columns, and pre-fills collection and tissue requirements.

I could explore the possibility of the string ``tissue`` doing something special in ``Part Search``, but I suspect I don't have the CPU to pull it off and it would cause mass confusion if I did.

**Additional context**

This is probably compounded by some collections (in and out of Arctos) calling all squishybits 'tissue' instead of naming parts.

**Priority**

?

**Comment by campmlc** [comment]
I agree this is a problem that needs a broad solution. MSB does not use "tissue" as part name unless we have no other info as we put different kinds of tissue in different vials that are explicitly searchable on "liver" or "heart,kidney". But users shouldn't have to know that. I like the idea of a general "tissue" search that would pull anything acceptable for "tissueness".  I also like a customizable search that we can add to our collections page or send to people with the needed values preset so no one has to guess. Eventually in some future reality we'd have individual collection pages with interfaces customized through the API - but we need something functional now, not in make-a-wish science fiction, which is where that exists at least for our institution.

**Comment by dustymc** [comment]
I don't think there's any part name : tissueness correlation in the data; suspect this is a documentation problem.

[temp_part_name_tiss.csv.zip](https://github.com/user-attachments/files/18286238/temp_part_name_tiss.csv.zip)

**Comment by camwebb** [comment]
We just discussed this issue as a group today (incl. @megulbranson). We were not sure where in Arctos the 'is tissue' search is looking. Scanning the Arctos code, I [see](https://github.com/ArctosDB/dev/blob/f3a8a0c1d47af0447afbf0ef024f209b634d9891/arctos_2.0/Reports/partusage.cfm#L12) that v2.0 had a `is_tissue` field in `ctspecimen_part_name` but using the SQL interface, this field is no longer there in more recent versions. So where is it specified that a particular part type is a tissue?  Thanks

**Comment by campmlc** [comment]
We had a funded protocol through GGBN to flag "tissueness" of any part based on their part preservation. A part with any history of preservation automatically excluded from "tissueness" - for example, fixation in formalin prior to transfer to 70% ethanol, would not show up as tissue. @dustymc couldn't this be integrated into our search protocols, and added as a search option? I thought it was. Or if not, we should be able to apply the same filter as we do to our GGBN resource?

**Comment by dustymc** [comment]
>. We were not sure where in Arctos the 'is tissue' search is looking.

https://handbook.arctosdb.org/documentation/parts.html#preservation

>funded protocol through GGBN to flag "tissueness"

No?!

**Comment by campmlc** [comment]
From our funded proposal to GGBN in 2017: 
"2) Standardization of controlled vocabulary for tissues. Currently data exported as a 'tissue' can be applied 
to frozen tissues, DNA extractions, whole organisms frozen and in ethanol, etc.'

**Comment by campmlc** [comment]
What is the current list of preservations that qualify for tissueness, and those that do not? And does this apply across all collection types? The list of parts under discussion from 2017 GGBN:  
https://docs.google.com/spreadsheets/d/1b9_JAx0LobFarYN-LYHVtvrZWi5o_p9T62qjrOwOwZo/edit?gid=845151152#gid=845151152
I understand we incorporated part preservation, but surely there must be some part intersection? Otherwise a mineral stored in 95% ethanol would qualify?

**Comment by mkoo** [comment]
didnt see a response-- there's a search specifically for tissues and it is exactly what you describe, simple toggle for yes|no has tissue
https://arctos.database.museum/search.cfm?sp=preset_tissues

this is determined in general if the cataloged item has anything that is available for molecular analysis so frozen tissue like liver, muscle etc or dried study skin

The[ list above is more or less accurate on what is a tissue](https://docs.google.com/spreadsheets/d/1b9_JAx0LobFarYN-LYHVtvrZWi5o_p9T62qjrOwOwZo/edit?gid=845151152#gid=845151152)
